filters/scheduler/lifo.go

```diff
@@ -248,37 +239,37 @@ func (l *lifoFilter) Response(ctx filters.FilterContext) {
 	response(scheduler.LIFOKey, ctx)
 }
 
-// Config returns the scheduler configuration for the given filter
-func (l *lifoGroupFilter) Config(r *scheduler.Registry) scheduler.Config {
-	return r.Config(l.key)
+func (l *lifoGroupFilter) Group() string {
+	return l.name
 }
 
-// SetStack binds the stack to the current filter context
-func (l *lifoGroupFilter) SetStack(s *scheduler.Stack) {
-	l.stack = s
+func (l *lifoGroupFilter) HasConfig() bool {
+	return l.hasConfig
```

Comment:
I guess you want to differentiate a chosen empty scheduler.Config with a not configured one, but do we actually need this?
As far as I see this would be also ok to test `l.config != scheduler.Config{}`.

Reply:
the difference is that it is also possible to set the defaults explicitly: `lifoGroup("groupname", 1, 0)`, and then it may occur that we don't get a config at all during the post-processing. This field tracks the difference.

Reply:
ok